tests/functional/adapter/microbatch/fixtures.py

```diff
@@ -0,0 +1,16 @@
+schema = """version: 2
+models:
+  - name: input_model
+
+  - name: microbatch_model
+    config:
+      persist_docs:
+        relation: True
+        columns: True
+    description: This is a microbatch model
+    columns:
```

Comment:
Added my own schema with column comments to supplement the included tests, since hitting comments originally broke my implementation despite passing the included functional tests.